packages/rc-tooltip/src/lib/rc-tooltip.spec.tsx

```diff
@@ -0,0 +1,101 @@
+import { RMWCProvider } from '@rmwc/provider';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { RCTooltip } from './rc-tooltip';
+
+describe('Tooltip', () => {
+  it('renders', async () => {
+    const { asFragment } = render(
+      <RCTooltip content="tooltip">
+        <span>test</span>
+      </RCTooltip>
+    );
+    await userEvent.hover(screen.getByText('test'));
+    expect(screen.getByText('tooltip')).toBeInTheDocument();
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('activateOn', () => {
+    render(
+      <RCTooltip content="tooltip" activateOn="click">
+        <span>test</span>
+      </RCTooltip>
+    );
+  });
+
+  it('showArrow', () => {
+    const { asFragment } = render(
+      <RCTooltip content="tooltip" showArrow>
+        <span>test</span>
+      </RCTooltip>
+    );
```

Comment:
Aha! It's because we don't dover on the tooltip I'm guessing. So we need to hover before rendering the snapshots.